packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.7+8
```

Comment:
I think we should make a new patch release (update version to 0.6.8). Using the +8 notation means the update is treated as a hotfix which in this case isn't really the case:

```suggestion
## 0.6.8
```